src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/TaxonImage.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" ?>
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.taxon_image.class%" shortName="TaxonImage">
+        <attribute name="validation_groups">sylius</attribute>
+
+        <collectionOperations>
+            <collectionOperation name="admin_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/admin/taxon-images</attribute>
+                <attribute name="normalization_context">
+                    <attribute name="groups">admin:taxon_image:read</attribute>
+                </attribute>
+            </collectionOperation>
```

Comment:
Quite reasonable doubts, because in the default Sylius we don't have much use for this endpoint. On the other hand, we provide a value that can be used and adjusted accordingly in end applications.

Reply:
IMO this endpoint is redundant :smiley:

Reply:
We've discussed it internally, so I'm resolving the conversation